src/library/impl/meta_reader/schema.h

```diff
@@ -48,7 +48,7 @@ namespace xlang::meta::reader
 
         auto Flags() const
         {
-            return TypeAttributes{ get_value<uint32_t>(0) };
+            return TypeAttributes{{ get_value<uint32_t>(0) }};
```

Comment:
Does this change to using the initalizer list ctor? Looking at the TypeAttributes class, I don't get why you made this change

Reply:
Hallway conversation: the code is basically initializing the base where that field actually resides. VC allows this but Clang (in strict mode) does not.

Reply:
This basically initializes the base type where the field actually resides. VC allows the former but Clang, in strict mode, does not.